core/domain/config_domain.py

```diff
@@ -235,7 +235,10 @@ def update_moderator_ids(committer_id, moderator_usernames):
     [])
 
 WHITELISTED_COLLECTION_EDITOR_USERNAMES = ConfigProperty(
-    'collection_editor_whitelist',
-    SET_OF_STRINGS_SCHEMA,
+    'collection_editor_whitelist', SET_OF_STRINGS_SCHEMA,
     'Names of users allowed to use the collection editor',
     [])
+
+AUTHORISED_QUERY_USERS = ConfigProperty(
```

Comment:
WHITELISTED_EMAIL_SENDERS, perhaps -- and then 'whitelisted_email_senders', 'Names of users allowed to send emails via the query interface.'